### PR TITLE
Update ansible/roles/rsyslog/defaults/main.yml

### PR DESCRIPTION
**Pull request by crafty-ua**
Extra check for ```rsyslog__syslog_srv_rr``` var

### CODE REVIEW

File: ansible/roles/rsyslog/defaults/main.yml [hunk 1]
```diff
@@ -298,7 +298,8 @@ rsyslog__syslog_srv_rr: '{{ q("dig", "_syslog._tcp." + rsyslog__domain + "./SRV"
 # record, logs will be snet to all of them at the same time.
 rsyslog__default_forward: '{{ (rsyslog__syslog_srv_rr | selectattr("target", "defined")
                                | list | sort)
-                              if ("NXDOMAIN" not in rsyslog__syslog_srv_rr and
+                              if (rsyslog__syslog_srv_rr and
+			          "NXDOMAIN" not in rsyslog__syslog_srv_rr and
```
Comment (SuperSandro2000) [inline ansible/roles/rsyslog/defaults/main.yml]:
```suggestion
                              "NXDOMAIN" not in rsyslog__syslog_srv_rr and
```
Tabs to spaces

Comment (crafty-ua) [reply ansible/roles/rsyslog/defaults/main.yml]: Thanks! Did not notice that github online editor uses tabs

**Comment by DouglasHeriot** [comment]
I just spent a bunch of time tracking down this same problem, and wrote the same pull request before seeing this.

In case it helps anyone else, applying the patch ansible/ansible#71666 made it much easier to track down the source of this error.

Where `ansible_domain` is `""`, `rsyslog__domain` is `""`, and `rsyslog__syslog_srv_rr` is `[]` - evaluating `rsyslog__default_forward` raises the error "list object has no element 0"

**Review by drybjed — APPROVED** [review]
I wasn't sure about the reasoning of this patch, but furter explanation by @DouglasHeriot made it clear, thanks! I'll backport this change to the stable releases as well.

**Comment by drybjed** [comment]
This PR was merged as https://github.com/debops/debops/commit/52302ac155b24c54fe03c783a47c06cc88ccd936, closing.

**Pull request closed without merging**